.github/workflows/publish.yml

```diff
@@ -18,29 +18,12 @@ jobs:
           node-version: 18
           registry-url: https://registry.npmjs.org/
           cache: npm
+      - run: npm install -g npm@latest
```

Comment:
In case other things break by globally installing it here, we could just move it down to where it was